OasisApi.ts

```diff
@@ -252,6 +254,7 @@ export async function settleHtlc(
     secret: string,
     settlementJWS: string,
     authorizationToken?: string,
+    smsApiToken?: string,
```

Comment:
Hmm. This will become a longer and longer list of arguments, if ever another service uses a different token type. And with the CRC OASIS API not taking an `authorizationToken`, the 4th argument to this function will always have to be set to `undefined`. I'd say we should change the tokens into an options object: 

```ts
tokens?: Partial<{
    authorizationToken: string,
    smsApiToken: string,
}>
```

Reply:
Also, when changed, this function signature has to be updated in `libswap` OASIS handlers' `OasisClient` interface.

Reply:
I'd say no. We should be the only ones using it, especially since OASIS EUR was shut down for a long time now.